packages/react-cookie/src/__tests__/withCookies-test.js

```diff
@@ -38,6 +38,10 @@ describe('withCookies(Component)', () => {
 
   describe('on the server', () => {
     it('provides the cookies', () => {
+      document.__defineGetter__('cookie', function() {
```

Comment:
Still need a fix here. Need a way to mock document.cookie and have HAS_DOCUMENT_COOKIE pick it up.